docs/gaia/join-testnet.md

```diff
@@ -103,6 +103,37 @@ curl https://raw.githubusercontent.com/cosmos/testnets/master/latest/genesis.jso
 Note we use the `latest` directory in the [testnets repo](https://github.com/cosmos/testnets)
 which contains details for the latest testnet. If you are connecting to a different testnet, ensure you get the right files.
 
+You now need to generate a `genesis tx` to generate a signed self-delegation:
+
+```bash
+gaiad gentx \
```

Comment:
Might want to mention that you need to `gaiacli keys add {{ .Name }}` before doing running `gentx`, otherwise it fails silently.